provider/src/main/java/com/synopsys/integration/alert/provider/blackduck/BlackDuckProperties.java

```diff
@@ -118,6 +118,11 @@ public BlackDuckHttpClient createBlackDuckHttpClient(IntLogger intLogger) throws
         return createBlackDuckHttpClient(intLogger, blackDuckServerConfig);
     }
 
+    public BlackDuckHttpClient createBlackDuckCacheClient(IntLogger intLogger) throws AlertException {
+        BlackDuckServerConfig blackDuckServerConfig = createBlackDuckServerConfig(intLogger);
+        return blackDuckServerConfig.createCacheHttpClient(intLogger);
+    }
+
     public BlackDuckHttpClient createBlackDuckHttpClient(IntLogger intLogger, BlackDuckServerConfig blackDuckServerConfig) {
         return blackDuckServerConfig.createBlackDuckHttpClient(intLogger);
     }
```

Comment:
Will we always want to use the cache client? If so, we should add a TODO for the second method here or deprecate it.

Reply:
No I don't think it makes sense to always use the cache client. We want to use it in the notification processing because we know there is a large of amount of duplicate requests but in the rest of the code it is likely to only add a memory cost with little to no benefit.